doc/Command-Reference.md

```diff
@@ -3559,6 +3587,67 @@ This command displays the details of Rx & Tx priority-flow-control (pfc) for all
   root@sonic:~# sonic-clear pfccounters
   ```
 
+**show pfc asymmetric**
+
+This command displays the status of asymmetric PFC for all interfaces or a given interface.
+
+- Usage:
+  ```
+  show pfc asymmetric
+  show pfc asymmetric <interface>
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show pfc asymmetric
+  
+  Interface    Asymmetric
+  -----------  ------------
+  Ethernet0    off
+  Ethernet2    off
+  Ethernet4    off
+  Ethernet6    off
+  Ethernet8    off
+  Ethernet10   off
+  Ethernet12   off
+  Ethernet14   off
+
+  admin@sonic:~$ show pfc asymmetric Ethernet0
+
+  Interface    Asymmetric
+  -----------  ------------
+  Ethernet0    off
+  ```
+
+**show pfc priority**
+
+This command displays the lossless priorities for all interfaces or a given interface.
+
+- Usage:
+  ```
+  show pfc priority
+  show pfc priority <interface>
```

Comment:
Combine these into one line, indicating that the <interface> parameter is optional, like this:

```
  show pfc priority [<interface>]
```

Reply:
@jleveque  Fixed